### PR TITLE
Add termux support for fish_config.

### PR DESCRIPTION
**Pull request by Liorst4**

### CODE REVIEW

File: share/tools/web_config/webconfig.py [hunk 1]
```diff
@@ -49,6 +49,10 @@ def is_wsl():
                 return True
     return False
 
+def is_termux():
+    """ Return whether we are running under the Terumx application for Android"""
+    return 'com.termux' in os.environ['PATH'] and 0 == os.system('which termux-open-url 1>&2 2>/dev/null')
```
Comment (krobelus) [inline share/tools/web_config/webconfig.py]: `which termux-open-url 1>&2 2>/dev/null` will send stdout to stderr and stderr to `/dev/null`.
If you want to discard both, swap the order of the redirection operators.

File: share/tools/web_config/webconfig.py [hunk 2]
```diff
@@ -49,6 +49,10 @@ def is_wsl():
                 return True
     return False
 
+def is_termux():
+    """ Return whether we are running under the Terumx application for Android"""
```
[the remark this answers is not in the document]
Comment (Liorst4) [reply share/tools/web_config/webconfig.py]: fixed

File: share/tools/web_config/webconfig.py (hunk 1 above)
Comment (Liorst4) [reply share/tools/web_config/webconfig.py]: fixed

Comment (faho) [reply share/tools/web_config/webconfig.py]: Also this check itself is really quite.... unclean, and invokes another shell *and* the `which` command, which doesn't have to be installed and should be discouraged (in shell scripts, use `command -v`).

I'd suggest `distutils.spawn.find_executable('termux-open-url')`, which returns a path if it finds it and None if it doesn't.

Comment (Liorst4) [reply share/tools/web_config/webconfig.py]: using `distutils` instead of `which`

**Comment by faho** [comment]
This looks much nicer now.

Merged, thanks!

**Pull request merged**